src/main.ts

```
@@ -21,8 +22,17 @@ Reflect.defineProperty(window, 'kuzzle', {
 Vue.use(BootstrapVue)
 Vue.use(BootstrapVueIcons)
 
+const analytics = new KeplerCompanion()
 const router = createRoutes(Vue.prototype.$log)
 
+router.afterEach(async (to, _) => {
+  await analytics.track({
```

You should not await here but only empty catch to not degrade the UX